updating/understanding-upgrade-channels-release.adoc

```diff
@@ -5,31 +5,11 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-In {product-title} 4.1, Red Hat introduced the concept of channels for recommending the appropriate release versions for cluster upgrades. By controlling the pace of upgrades, these upgrade channels allow you to choose an upgrade strategy. Upgrade channels are tied to a minor version of {product-title}. For instance, {product-title} 4.9 upgrade channels recommend upgrades to 4.9 and upgrades within 4.9. They also recommend upgrades within 4.8 and from 4.8 to 4.9, to allow clusters on 4.8 to eventually upgrade to 4.9. They do not recommend upgrades to 4.10 or later releases. This strategy ensures that administrators explicitly decide to upgrade to the next minor version of {product-title}.
+Upgrade channels enable cluster administrators to choose an upgrade strategy. {product-title} 4 releases are sequential. Updates are decoupled from releases which enables you to create dynamic upgrade paths.
```

Comment:
Not clear to me what these later two sentences are trying to convey.  Before, we were attempting to convey that `fast-4.9` included 4.8->4.8, 4.8->4.9, and 4.9->4.9 updates, but would not include *->4.10 updates, and that seems to have gotten lost.  There is some `previous minor version releases` discussion in the channel sections, but nothing in those sections about the update edges, just discussion of the update nodes.  We probably want to update the channel docs to explicitly distinguish edges (update recommendations) from nodes (releases), because [phased rollouts][1], which we'll revive at some point, will promote nodes to all supported channels (fast and later) at the same time, and only delay the edges while we cook collecting more feedback.  The thing that is true both now and in the hypothetical future is that the update recommendation edges are delayed between fast and the later channels. 

[1]: https://github.com/openshift/enhancements/pull/427/files#diff-94d69b3d9a7f7be29e486c6ea4df58d3b112f0d3dc24a3a2a2496ae53f9b9df1R65